libs/shared/l10n/src/lib/localizer/localizer.rsc.factory.spec.ts

```diff
@@ -70,12 +70,7 @@ describe('LocalizerRscFactory', () => {
       }).compile();
       const emptyLocalizer = module.get(LocalizerRscFactory);
 
-      expect.assertions(1);
-      try {
-        emptyLocalizer.createLocalizerRsc('en');
-      } catch (error) {
-        expect(error).toBeDefined();
-      }
+      expect(() => emptyLocalizer.createLocalizerRsc('en')).toThrow();
```

Comment:
Q: When using `.toThrow()` should we be checking for specific Error class or message?
```sh
git grep -Pn "\.toThrow\(\)" -- apps libs | grep -v "\.not\.toThrow()" | grep -v "// expect()"

libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.repository.in.spec.ts:46:      expect(createPaypalCustomer(kyselyDb, paypalCustomer)).rejects.toThrow();
libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.repository.in.spec.ts:71:      ).rejects.toThrow();
libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.repository.in.spec.ts:180:      ).rejects.toThrow();
libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.repository.in.spec.ts:216:      ).rejects.toThrow();
libs/payments/stripe/src/lib/accountCustomer/accountCustomer.repository.in.spec.ts:49:      ).rejects.toThrow();
libs/payments/stripe/src/lib/accountCustomer/accountCustomer.repository.in.spec.ts:86:      ).rejects.toThrow();
libs/payments/stripe/src/lib/accountCustomer/accountCustomer.repository.in.spec.ts:94:      ).rejects.toThrow();
libs/payments/stripe/src/lib/accountCustomer/accountCustomer.repository.in.spec.ts:146:      ).rejects.toThrow();
libs/shared/l10n/src/lib/localizer/localizer.rsc.factory.spec.ts:73:      expect(() => emptyLocalizer.createLocalizerRsc('en')).toThrow();
```

versus:

```sh
git grep -Pn "\.toThrow\(" -- apps libs | grep -v "\.not\.toThrow()" | grep -v "// expect()" | grep -v "\.toThrow()" | grep -Ev "\.toThrow\($"

libs/payments/cart/src/lib/cart.service.spec.ts:323:      ).rejects.toThrow(Error);
libs/payments/cart/src/lib/cart.service.spec.ts:341:      ).rejects.toThrow(Error);
libs/payments/currency/src/lib/currency.manager.spec.ts:42:      ).toThrow(CurrencyCodeInvalidError);
libs/payments/currency/src/lib/currency.manager.spec.ts:53:      ).toThrow(CountryCodeInvalidError);
libs/payments/currency/src/lib/currency.manager.spec.ts:64:      ).toThrow(CurrencyCountryMismatchError);
libs/payments/paypal/src/lib/paypal.manager.spec.ts:444:      }).toThrow(AmountExceedsPayPalCharLimitError);
libs/payments/paypal/src/lib/paypalCustomer/paypalCustomer.repository.in.spec.ts:154:      ).rejects.toThrow(PaypalCustomerNotUpdatedError);
libs/payments/stripe/src/lib/accountCustomer/accountCustomer.repository.in.spec.ts:124:      ).rejects.toThrow(AccountCustomerNotUpdatedError);
libs/shared/cloud-tasks/src/lib/account-tasks.spec.ts:125:      ).rejects.toThrow('BOOM');
libs/shared/contentful/src/lib/contentful.client.spec.ts:123:      ).rejects.toThrow(new ContentfulError([error]));
libs/shared/sentry/src/lib/config-builder.spec.ts:78:    }).toThrow('sentry.dsn not specified. sentry disabled.');
libs/shared/sentry/src/lib/config-builder.spec.ts:103:    }).toThrow('config missing either release or version.');
libs/shared/sentry/src/lib/config-builder.spec.ts:114:    }).toThrow('sentry.sampleRate');
```